### PR TITLE
CTCTRADERS-2418: remove guard for current users and then change type for Total Gross Mass

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

**Review by sawan07 — APPROVED** [review]

File: test/models/userAnswerScenarios/Scenario1.scala [hunk 1]
```diff
@@ -204,7 +204,7 @@ case object Scenario1 extends UserAnswerScenario {
      * Item Details section - Item Two
      * */
     .unsafeSetVal(pages.ItemDescriptionPage(secondGoodItem))("ItemTwosDescription")
-    .unsafeSetVal(pages.ItemTotalGrossMassPage(secondGoodItem))("25001")
+    .unsafeSetVal(pages.ItemTotalGrossMassPage(secondGoodItem))(25000.000)
```
Comment (ericohwotu) [inline test/models/userAnswerScenarios/Scenario1.scala:207]: Could you please make this number different

**Review by ericohwotu — APPROVED** [review]

File: test/models/journeyDomain/JourneyDomainSpec.scala [hunk 2]
```diff
@@ -37,6 +38,53 @@ class JourneyDomainSpec extends SpecBase with GeneratorSpec with UserAnswersGene
...
+              val itemSection1       = itemsSection.copy(itemDetails = itemDetailsSection.copy(itemTotalGrossMass = "100.123"))
+              val itemSection2       = itemsSection.copy(itemDetails = itemDetailsSection.copy(itemTotalGrossMass = "200.246"))
+
+              val result = ItemSections(NonEmptyList(itemSection1, List(itemSection2))).totalGrossMassFormatted
+
+              result mustBe "300.369"
+          }
+        }
+      }
+//  "JourneyDomain" - {
```
Comment (ericohwotu) [inline test/models/journeyDomain/JourneyDomainSpec.scala:55]: Why does it look like you commented this out?

Comment (kenneth-rayner) [reply test/models/journeyDomain/JourneyDomainSpec.scala:55]: Was commented out in April  not scope on this ticket but will need looked at

Comment (kenneth-rayner) [reply test/models/journeyDomain/JourneyDomainSpec.scala]: Removed

**Pull request merged**